react/features/subtitles/middleware.js

```diff
@@ -115,7 +116,7 @@ function _endpointMessageReceived({ dispatch, getState }, next, action) {
                 newTranscriptMessage));
 
         } else if (json.type === JSON_TYPE_TRANSCRIPTION_RESULT
-                && !translationLanguage) {
+        && i18next.language === translationLanguage) {
```

Comment:
Why is this check performed?

Reply:
There is an edge case when a user is selecting the same language as the transcription. In this case both source language and destination language is **en** and the google translation api returning an error that the two languages are equal. By doing this check the user can display the transcription, and i don't have the remove the source language for the list of available languages and transcription is also available.